run_canopy_slurm.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -l
+#SBATCH --partition=bigmem             # big memory node
+#SBATCH --job-name=canopy-app          # name the job
+#SBATCH --output=canopy-%j.out         # write stdout to named file
+#SBATCH --error=canopy-%j.err          # write stdout to named file
+#SBATCH --time=0-02:00:00              # Run for max of 00 hrs, 10 mins, 00 secs
+#SBATCH --nodes=1                      # Request N nodes
+#SBATCH --exclude=hop006,hop010,hop011 # Exclude some nodes (optional)
+#SBATCH --ntasks=1                     # Request n tasks
+#SBATCH --mem-per-cpu=60GB             # Request nGB RAM per core
```

Comment:
@angehung5 A single node/task with your suggestion of 60GB memory may not work, please test this example.

I don't think we really need to increase ntasks because its not parallel code.

From my experience using a single node/task requires like about 256 GB memory.